webapp/app.py

```diff
@@ -0,0 +1,31 @@
+"""
+A Flask application for jp.ubuntu.com
+"""
+
+import flask
+from werkzeug.contrib.fixers import ProxyFix
+from werkzeug.debug import DebuggedApplication
+
+from webapp.blueprint import jp_website_blueprint
+from webapp.handlers import set_handlers
+
+
+def create_app(testing=False):
```

Comment:
I assume you copied this pattern [from snapcraft.io](https://github.com/canonical-websites/snapcraft.io/blob/a591f87fa2a7ad754b9d3897921f0a6da9c4cb26/webapp/app.py#L24), but I'm really not a fan of it. Unless you strongly object, could you please remove the `create_app` wrapper and just use the `app` directly?

Reply:
Sure. No objections

Reply:
Updated it. Now using app directly in the entrypoint. Also removed the unused `wsgi.py`.
Also not sure about talisker.